docker-compose-zero-trust.yml

```diff
@@ -0,0 +1,1136 @@
+    user: 2002:2001
+    volumes:
+    - type: volume
+      source: edgex-init
+      target: /edgex-init
+      read_only: true
+      volume: {}
+    - type: bind
+      source: /etc/localtime
+      target: /etc/localtime
+      read_only: true
+      bind:
+        create_host_path: true
+    - type: bind
+      source: /tmp/edgex/secrets/support-notifications
+      target: /tmp/edgex/secrets/support-notifications
+      read_only: true
+      bind:
+        selinux: z
+        create_host_path: true
+  support-scheduler:
+    command:
+    - /support-scheduler
+    - -cp=consul.http://edgex-core-consul:8500
+    - --registry
+    container_name: edgex-support-scheduler
+    depends_on:
+      consul:
+        condition: service_started
+      core-common-config-bootstrapper:
+        condition: service_started
+      database:
+        condition: service_started
+      security-bootstrapper:
+        condition: service_started
+      security-secretstore-setup:
+        condition: service_started
+    entrypoint:
+    - /edgex-init/ready_to_run_wait_install.sh
+    environment:
+      EDGEX_SECURITY_SECRET_STORE: "true"
+      INTERVALACTIONS_SCRUBAGED_HOST: edgex-core-data
+      INTERVALACTIONS_SCRUBPUSHED_HOST: edgex-core-data
+      SECRETSTORE_HOST: edgex-vault
+      SERVICE_HOST: support-scheduler.edgex.ziti
+      SERVICE_PORT: 80
+      STAGEGATE_BOOTSTRAPPER_HOST: edgex-security-bootstrapper
+      STAGEGATE_BOOTSTRAPPER_STARTPORT: "54321"
+      STAGEGATE_DATABASE_HOST: edgex-redis
+      STAGEGATE_DATABASE_PORT: "6379"
+      STAGEGATE_DATABASE_READYPORT: "6379"
+      STAGEGATE_PROXYSETUP_READYPORT: "54325"
+      STAGEGATE_READY_TORUNPORT: "54329"
+      STAGEGATE_REGISTRY_HOST: edgex-core-consul
+      STAGEGATE_REGISTRY_PORT: "8500"
+      STAGEGATE_REGISTRY_READYPORT: "54324"
+      STAGEGATE_SECRETSTORESETUP_HOST: edgex-security-secretstore-setup
+      STAGEGATE_SECRETSTORESETUP_TOKENS_READYPORT: "54322"
+      STAGEGATE_WAITFOR_TIMEOUT: 60s
+    hostname: edgex-support-scheduler
+    image: nexus3.edgexfoundry.org:10004/support-scheduler:latest
+    networks:
+      edgex-network: null
+    read_only: true
+    restart: always
+    security_opt:
+    - no-new-privileges:true
+    user: 2002:2001
+    volumes:
+    - type: volume
+      source: edgex-init
+      target: /edgex-init
+      read_only: true
+      volume: {}
+    - type: bind
+      source: /etc/localtime
+      target: /etc/localtime
+      read_only: true
+      bind:
+        create_host_path: true
+    - type: bind
+      source: /tmp/edgex/secrets/support-scheduler
+      target: /tmp/edgex/secrets/support-scheduler
+      read_only: true
+      bind:
+        selinux: z
+        create_host_path: true
+  ui:
+    command:
+      - "./edgex-ui-server"
+      - "--configDir=res/docker"
+    container_name: edgex-ui-go
+    environment:
+      EDGEX_SECURITY_SECRET_STORE: "true"
+      SERVICE_HOST: 0.0.0.0
+      _SERVICE_PORT: 4000
```

Comment:
Why the underscore? 4000 is already the Port value in the config file so no need for an override.

Reply:
junk. will remove